ViewModels/Cart.cs

```diff
@@ -9,7 +9,10 @@ namespace BangazonWeb.ViewModels
   public class CartView : BaseViewModel
   {
     public float TotalPrice { get; set; }
+    public Order Order { get; set; }
+    public PaymentType PaymentType { get; set; }
     public IEnumerable<Product> ActiveProducts { get; set; }
+    public IEnumerable<Product> LineItems { get; set; }
```

Comment:
Again, what's the difference between ActiveProducts and LineItems? I think we only need one of these